### DISCUSSION TITLE
[BUG]Why is the If activity in designer 3.1.3 only having the Condition attribute and not the Then and Else attributes?

### DISCUSSION THREAD

**Discussion by shxylgithub** [elsa-workflows/elsa-core#5627] [Q&A]
Why is the If activity in designer 3.1.3 only having the Condition attribute and not the Then and Else attributes?
![If](https://github.com/elsa-workflows/elsa-core/assets/171897414/61f84d72-8534-4384-bf0c-58b7acf0a9e3)

**Comment by sfmskywalker** [comment]
The Then and Else properties are displayed directly on the activity itself. The way this activity works is as follows:

- If the condition evaluates to `true`, the activity assigned to the `Then` property is scheduled for execution.
- If the condition evaluates to `false`, the activity assigned to the `Else` property is scheduled for execution.

**Reply by shxylgithub** [reply]
Hello teacher
The Then and Else properties are directly displayed on the If activity itself. How can we set the plan to execute the activity assigned to the Then property and the plan to execute the activity assigned to the Else property in the designer? Thank you!
![IfSetting](https://github.com/elsa-workflows/elsa-core/assets/171897414/10d3df5b-b3b1-457a-87cb-d38fd7a590fa)

**Comment by shxylgithub** [comment]
Hello teacher:
The Then and Else properties are directly displayed on the If activity itself. How can we set the plan to execute the activity assigned to the Then property and the plan to execute the activity assigned to the Else property in the designer? Thank you!






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "elsa-workflows/elsa-core"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年6月18日(星期二) 晚上9:10
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [elsa-workflows/elsa-core] [BUG]Why is the If activity in designer 3.1.3 only having the Condition attribute and not the Then and Else attributes? (Discussion #5627)





 
The Then and Else properties are displayed directly on the activity itself. The way this activity works is as follows:
  
If the condition evaluates to true, the activity assigned to the Then property is scheduled for execution.
 
If the condition evaluates to false, the activity assigned to the Else property is scheduled for execution.

**Comment by shxylgithub** [comment]
Why does the If activity in Elsa 3.1.3 designer only have the Condition attribute and not the Then and Else attributes? Elsa 3.1.3 designer seems to have an issue where Then and Else properties are not visible in the designer interface.
